### ISSUE TITLE
Insufficient contrast ratio on links (claim by Chrome Lighthouse)

### ISSUE THREAD

**Issue by SirDaev** [SirDaev/langits#16]
Verify whether the link color vs. background color has sufficient contrast ratio or not, and change the site's blue color if needed.

![image](https://user-images.githubusercontent.com/12941979/233894148-75ab39ae-72ab-4ae4-9257-5cd06f938121.png)
